### ISSUE TITLE
The app uses the user's local time zone instead of the group's time zone

### ISSUE THREAD

**Issue by jamescmartinez** [ThumbWorks/sf-ios#151]
While setting up the [Chicacoland iOS Coffee](https://coffeecoffeecoffee.coffee/chicagoland-ios-coffee) group, @joshdholtz discovered a time zone discrepancy between the app and the site.

The site uses the group's time zone instead of the user's time zone. The app uses the user's time zone and does not use the group's defined time zone. 

Here's what the data looks like:
* backend database: **12:30PM UTC**
* group time zone: `America/Chicago`
* frontend web: **7:30AM**
* API: **12:30PM UTC**
* iOS: depending on where the iPhone is located, you'd see `5:30AM` on the west coast and `8:30AM` on the east coast.

We should discuss a solution that is consistent across both web and mobile.

I'm personally leaning toward showing events in the group's time zone instead of the user's time zone. The counter-argument would be if you want to join remotely for whatever reason, you'd need your local time zone instead of the group's time zone. With that said, both Meetup and Eventbrite use the approach the site uses now, displaying events relative to the event location rather than the user's current location.

What do you all think?

**Comment by mcritz** [comment]
Server should be the source of truth.

**Comment by jamescmartinez** [comment]
It is now, but both platforms just handle dates differently. This might be more of a UX question than it is a data question. I think it comes down to:

*Do we want users to see dates in their local time zone or in the time zone of the group?*

Once we figure that out we can figure out the best way to get the right data to the app so we're consistent across the board no matter which UX approach we choose.

**Comment by joshdholtz** [comment]
I _think_ seeing in timezone of the event makes the most since. If I looked at the app before I came to SF I would expect to be able to see it in Pacific timezone (otherwise it would probably mess up my planning 😊 )

**Comment by rodericj** [comment]
I agree with what josh said.

**Comment by jamescmartinez** [comment]
That seems to be others' reactions as well from when Meetup switched from the user timezone to the event timezone. http://www.discussmeetup.com/forum/techy-issues-and-workarounds/time-zones-in-new-meetup/

In this case, the iOS app can convert to the group's time zone. When the API returns a group, it returns a `time_zone`: https://github.com/jamescmartinez/coffee/blob/master/docs/openapi.yaml#L74

**Comment by mcritz** [comment]
Agreed. This is a good enhancement to get done properly before dubdub
